### ISSUE TITLE
[Crash] ... Minecraft Crashing from server 

### ISSUE THREAD

**Issue by RastTattoo** [SuperMartijn642/Rechiseled#110]
**Version Info**
- Minecraft, ... 1.21.1
- Rechiseled, ... 1.1.6
- Fusion, ...

**What mod loader are you using?**: Fabric/Forge/NeoForge/Quilt
Forge
**Steps to Reproduce**
...

**Crash report (~/logs/latest.log)**
...Error: java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because "com.supermartijn642.rechiseled.chiseling.ChiselingRecipes.chiselingRecipes" is null

**Comment by SuperMartijn642** [comment]
This is caused by I by a mistake in Forge, one that I actually introduced in a pull request I submitted 🙃
I have submitted another PR to fix my previous mistake here https://github.com/MinecraftForge/MinecraftForge/pull/10171.
The issue should go away once that is merged.

**Comment by SuperMartijn642** [comment]
This has been fixed now in Forge 52.0.29.
Thank you for reporting the issue!
